src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -155,7 +156,11 @@ public function pFileWithoutNamespace(FileWithoutNamespace $fileWithoutNamespace
 
     protected function p(Node $node, $parentFormatPreserved = false): string
     {
-        $content = parent::p($node, $parentFormatPreserved);
+        if ($node instanceof Expr && $node instanceof VirtualNode && method_exists($node, 'getExpr')) {
```

Comment:
Some comment here to explain it would be helpful. 

```suggestion
        if ($node instanceof VirtualNode && method_exists($node, 'getExpr')) {
```

Reply:
`Expr` instance check instance is needed, `VirtualNode` can be `UnreachableStatementNode` which got Stmt

Reply:
I added comment why Expr and VirtualNode check is needed https://github.com/rectorphp/rector-src/pull/4305/commits/63e1744ba0968c84ae5da6a6bc365bc8d1e05501